extensions/console-notification.crd.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consolenotifications.console.openshift.io
+  annotations:
+    displayName: ConsoleNotification
```

Comment:
Think we should rename this to `ConsoleNotificationBanner`.  Just for clarity.  This is not the way `Notification` is typically used & understood.

Reply:
I will likely add `Banner` to https://github.com/openshift/console-operator/pull/229/files#r285667400

Reply:
I'd suggest `ConsoleBanner` instead.  `ConsoleNotificationBanner` is getting pretty long.